src/DataValidation.py

```diff
@@ -192,6 +192,67 @@ def checkItemsThatShouldBeRequired():
                     if item["name"] in region_requires:
                         raise ValidationError("Item %s is required by region %s, but the item is not marked as progression." % (item["name"], region_name))
 
+    @staticmethod
+    def checkIfEnoughItemsForValue():
+        values_available = {}
+        values_requested = {}
+
+        # get all the available values with total count
+        for item in DataValidation.item_table:
+            # if the item is already progression, no need to check
+            if not "progression" in item or not item["progression"]:
+                continue
+
+            # progression_skip_balancing is also progression, so no check needed
+            if not "progression_skip_balancing" in item or not item["progression_skip_balancing"]:
+                continue

```

Comment:
Items either have one of the other.  You're eliminating both independently, so no items will make it past the second check.

Replace both of these with the following:
```py
if not item.get("progression") and not item.get("progression_skip_balancing"):
    continue
```

Reply:
I went with the De Morgan law of boolan logic to invert it, I'll modify it later when I got time to test it properly